doc/scope.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if scope}}
```

Comment:
Purpose of all custom templates and helpers.js is just to support custom @async tag. 
I think, that they can be packaged as plugin for [dmd](https://www.npmjs.com/package/dmd).